app/controllers/reports/subscription_reports_for_project_owners_controller.rb

```diff
@@ -0,0 +1,24 @@
+require 'csv'
+
+class Reports::SubscriptionReportsForProjectOwnersController < Reports::BaseController
+  def index
+    if policy(Project.find(params[:project_id])).update?
+      @report = end_of_association_chain.to_csv
+      respond_to do |format|
+        format.csv { send_data @report, filename: 'apoiadores.csv' }
+      end
+    else
+      redirect_to root_path
+    end
+  end
+
+  private
+
+  def end_of_association_chain
+    conditions = { project_id: params[:project_id] }
```

Comment:
Why not put this directly on the `where` clause? IMHO, it turns the call more readable.